src/main/include/log4cxx/rolling/gzcompressaction.h

```diff
@@ -52,6 +52,17 @@ class GZCompressAction : public Action
 		 */
 		bool execute(log4cxx::helpers::Pool& pool) const override;
 
+		/**
+		 * Set to true to throw an IOException on a fork failure.  By default, this
+		 * is true.  When an IOException is thrown, this will automatically cause the
+		 * error handler to be called(which is the recommended way of handling this
+		 * problem).  By setting this to true, the GZCompressAction effectively
```

Comment:
Did you mean "By setting this to false, ..."?

Reply:
whoops, thanks for noticing!  fixed.